x/ammswap/handler.go

```diff
@@ -452,5 +456,5 @@ var (
 // mulAndQuo returns a * b / c
 func mulAndQuo(a, b, c sdk.Dec) sdk.Dec {
 	a = a.Mul(auxiliaryDec)
-	return a.Mul(b).Quo(c).Quo(auxiliaryDec)
+	return a.Mul(b).Quo(c).QuoTruncate(auxiliaryDec)
```

Comment:
it is suggested to change all of mul or quo to MulTruncate or QuoTruncate in ammswap module.